src/components/Header.js

```diff
@@ -0,0 +1,9 @@
+const Header = () => {
+  return <div><h1 class="mainheader">Welcome to NC News</h1>
+  <h2 class="sub-header"> Latest Articles:</h2>
+  
+  </div>;
```

Comment:
Can you think of a more semantic tag than a `div` to use to divide out the header component? 👀